src/components/Error/ErrorBoundary.tsx

```diff
@@ -1,4 +1,10 @@
-import { DEFAULT_ERROR_ACTION, DEFAULT_ERROR_HEADER, WidgetError } from 'errors'
+import {
+  DEFAULT_ERROR_ACTION,
```

Comment:
These defaults are only used in this file. Could you just define them here, to avoid importing them?